pkg/certificates/triple/cert/cert.go

```diff
@@ -161,8 +161,10 @@ func GenerateSelfSignedCertKeyWithFixtures(host string, alternateIPs []net.IP, a
 	certFixturePath := path.Join(fixtureDirectory, baseName+".crt")
 	keyFixturePath := path.Join(fixtureDirectory, baseName+".key")
 	if len(fixtureDirectory) > 0 {
+		// #nosec No risk for path injection. Used internally to read cert files
```

Comment:
I think this function is not used. Too be exact the function which calls this is not used anymore.

Reply:
I removed both functions